spec/jobs/nb_person_push_address_job_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe NbPersonPushAddressJob, type: :job do
+  describe "#perform" do
+    it "calls NationBuilder with correct args" do
+      args = { attributes: { email: "user@example.com", registered_address: { city: "Keene" } } }
+      expect(Integration::NationBuilder).to receive(:create_or_update_person).with(args)
+      NbPersonPushAddressJob.new.perform('user@example.com', city: 'Keene')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.